python/README.txt

```diff
@@ -0,0 +1,95 @@
+nGraph Compiler stack
```

Comment:
Why is there a `README.txt` file here?
Let's remove it and keep `.md` only.

Reply:
Why is because the correct way to refer to the "Intel Nervana NNP" should have the symbols; the least necessary builder was breaking due to our correctness.